bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Collections.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace ElectionGuard
+{
+    using NativeLinkedList = NativeInterface.LinkedList.LinkedListHandle;
+
+    /// <Summary>
+    /// A singly-linked list data structure composed of strings for keys and strings for values.
+    /// </Summary>
+    public class LinkedList : DisposableBase
+    {
+        /// <Summary>
+        /// Get the count of elements in the list.
+        /// </Summary>
+        public unsafe ulong Count
+        {
+            get
+            {
+                return NativeInterface.LinkedList.GetCount(Handle);
+            }
+        }
+
+        internal unsafe NativeLinkedList Handle;
+
+        unsafe internal LinkedList(NativeLinkedList handle)
+        {
+            Handle = handle;
+        }
+
+        unsafe public LinkedList()
+        {
+            var status = NativeInterface.LinkedList.New(out Handle);
+            status.ThrowIfError();
```

Comment:
throwing in the constructor is preferable in the short term to creating a managed class that has no unmanaged allocation.